lib_test/dune

```diff
@@ -1,6 +1,6 @@
 (executables
  (names benchmark test stress)
- (libraries lwt cstruct-lwt mirage-block-unix cstruct io-page io-page-unix
+ (libraries lwt cstruct-lwt mirage-block-unix cstruct io-page io-page.unix
```

Comment:
That should just be `io-page`, right? 

Because of: https://github.com/mirage/io-page/blob/master/lib/io_page_unix.ml

Reply:
Yes, we can do another release if you want.